apps/aeutils/priv/epoch_config_schema.json

```
@@ -241,6 +241,11 @@
                             "description" : "Number of acceptors in internal pool",
                             "type" : "integer",
                             "default" : 10
+                        },
+                        "debug_endpoints" : {
+                            "description" : "Enable (true) debug api. Disabled (false) by default. Debug endpoints are defined by a tag in swagger.yaml config",
```

A minor readability improvement.

```suggestion
                            "description" : "Enable (true) debug api. Disabled (false) by default. Debug endpoints are defined by a tag in the Swagger API schema",
```